### PR TITLE
Fix ik matlab communication

### PR DESCRIPTION
**Pull request by yimingyanged**
@patmarion  Sorry for the late reply. The issue in #235 #236 is because pushToMatlab is used in a conflict way. It is used to indicate both 'push to matlab or not' and 'use matlab or exotica to solve IK'. I added a new variable to select the planning mode (for now, drake or exotica). The pushToMatlab var is now only used to indicate whether push things to matlab.

### CODE REVIEW

File: src/python/director/startup.py [hunk 1]
```diff
@@ -1086,12 +1086,6 @@ def stop(self):
     #ikPlanner.fixedBaseArm = True
     #showImageOverlay()
 
-
-if 'exo' in drcargs.args():
```
Comment (wxmerkt) [inline src/python/director/startup.py]: I believe this also requires a change to https://github.com/openhumanoids/director/blob/fix-ik-matlab-communication/src/python/director/drcargs.py#L77 or replace the functionality here to set ``self.ikPlanner.planningMode = 'exotica'`` if exo is in drcargs.args()

**Comment by patmarion** [comment]
could you change this so that only the `addPose` function checks the  `self.pushToMatlab` variable?  The other functions should not use `pushToMatlab`, the variable was only ever meant to switch behavior or the `addPose` function.

**Comment by yimingyanged** [comment]
@patmarion I have removed pushToMatlab in runIK and runIKTraj

**Comment by patmarion** [comment]
thanks!

**Pull request merged**